src/main/java/seedu/address/model/order/OrderManager.java

```diff
@@ -63,6 +63,13 @@ public void setOrder(List<OrderItem> orderItems) {
         saveChanges();
     }
 
+    /**
+     * Clears the contents of orderHistory.
+     */
+    public void clearOrderHistory() {
+        this.orderHistory.clear();
+    }
```

Comment:
If called by itself, the assumption that the head of orderHistory is equal to order is no longer true.